doc/en/user/source/community/gsr/dynamic_map_layer/layers-dynamicmaplayer.html

```diff
@@ -0,0 +1,77 @@
+<!-- Esri®, ArcGIS® and ArcGIS Online®  are trademarks, registered trademarks, or service marks of Esri in the United States, the European Community, or certain other jurisdictions. Other companies and products mentioned may be trademarks of their respective owners.-->
```

Comment:
I think technically xml docs need to start with the DOCTYPE line, comments can come after :P

Reply:
fixed